services/web-app/components/full-page-error/full-page-error.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Column, Grid } from '@carbon/react'
+import PropTypes from 'prop-types'
+
+import styles from './full-page-error.module.scss'
+
+export const FullPageError = ({ title, subtitle, description, content }) => {
+  return (
+    <Grid>
+      <Column className={styles.column} sm={4} md={8} lg={{ start: 6, span: 6 }}>
+        <h1 className={styles.title}>{title}</h1>
+        <p className={styles.subtitle}>{subtitle}</p>
+        <p className={styles.description}>{description}</p>
+        <div className={styles.content}>{content}</div>
```

Comment:
Do we need description and content here? I think we probably just need one of those and need to add an optional link at the bottom:
<img width="729" alt="image" src="https://user-images.githubusercontent.com/40550942/180237536-95fb39a4-8f29-47f8-b761-022e42163fbf.png">
<img width="526" alt="image" src="https://user-images.githubusercontent.com/40550942/180237663-b035bd80-ab0a-43c1-8faa-a84dbccf6791.png">